db/migrations/84_drop_column_items_items_id.sql

```diff
@@ -0,0 +1,48 @@
+-- +migrate Up
+DELETE `items_items` FROM `items_items`
+    JOIN (
+        SELECT `id`, ROW_NUMBER() OVER (PARTITION BY `items_items`.`parent_item_id`, `items_items`.`child_item_id` ORDER BY `items_items`.`child_order`) AS number
+        FROM `items_items`
+            JOIN (
+                SELECT `parent_item_id`, `child_item_id`, count(*) AS cnt
+                FROM `items_items`
+                GROUP BY `parent_item_id`, `child_item_id` HAVING cnt > 1
+            ) AS `duplicates` -- not unique (parent_item_id, child_item_id) pairs
+                ON `duplicates`.`parent_item_id` = `items_items`.`parent_item_id` AND
+                 `duplicates`.`child_item_id` = `items_items`.`child_item_id`
+    ) AS `duplicated_rows` -- ids of duplicated row with row numbers within (parent_item_id, child_item_id) group, ordered by child_order
+        ON `duplicated_rows`.`id` = `items_items`.`id` AND `duplicated_rows`.`number` > 1;  -- remove duplicates keeping first rows
```

Comment:
Seems to be a very complex version of the simple: 
```sql
DELETE items_items FROM items_items 
JOIN items_items orig USING (parent_item_id, child_item_id) 
WHERE items_items.id > orig.id;
```
😁

Reply:
Mine orders by `child_order`

Reply:
```sql
DELETE items_items FROM items_items 
JOIN items_items orig USING (parent_item_id, child_item_id) 
WHERE items_items.order > orig.order;
```
Still much much less complex than yours :-P

Reply:
(you can leave it like that if you want, but I just saw it twice with something very complex)

Reply:
fixed